django_project/feti/views/share.py

```diff
@@ -27,15 +31,16 @@
     ApiCampus
 )
 from feti.api_views.common_search import CommonSearch
-
-
+from feti.serializers.favorite_serializer import FavoritePDFSerializer
+from haystack.inputs import Clean, Exact
```

Comment:
please put this import after django import

http://kartoza.com/en/coding-standards/